README.md

```diff
@@ -56,6 +56,51 @@ We provide the following ways for you to use xrefcheck:
 
 If none of those are suitable for you, please open an issue!
 
+If you want some external links to not be verified, you can use one of the following ways to ignore that links:
```

Comment:
To avoid making README too big, let's put this into a separate section and hide under a spoiler, something like this:

```markdown
### Special functionality

<details>
  <summary>Ignoring external links</summary>
  Text goes here
</details>
```